src/models/model.cpp

```diff
@@ -537,6 +537,7 @@ std::unique_ptr<OrtValue> Model::ExpandInputs(std::unique_ptr<OrtValue>& input,
 }
 
 void Model::GetMaxBatchSizeFromGeneratorParams(const GeneratorParams& params) {
```

Comment:
>GetMaxBatchSizeFromGeneratorParams [](http://example.com/codeflow?start=12&length=34)

why is it commented out?

Reply:
Whoops, that wasn't supposed to be merged.